### DISCUSSION TITLE
Can you host "hidden" studies through pushkin?

### DISCUSSION THREAD

**Discussion by raryskin** [l3atbc-datadog/pushkin#170] [General] [answered]
I'd like to run a study on AMT and then potentially switch it to become a publicly available task on my (future) online lab. Is there an easy way to deploy a task with pushkin but have it not appear on the main page? Sorry if this is in the documentation and I missed it. It would be great if I could easily toggle whether an experiment is available via direct link only or if it appears on the main page. Is that currently an option/how would I do that? Thanks!

**Comment by jkhartshorne** [comment, marked as answer]
@raryskin -- not yet. But that's a pretty good idea, and it shouldn't be too hard to set up. I've been planning on setting up a method of archiving experiments, so this would be pretty similar. I'll put it on the queue. Realistically, it'll probably be a few weeks before we have a release with this functionality.
